### DISCUSSION TITLE
Specify input or role for snapshots

### DISCUSSION THREAD

**Discussion by chokays** [blakeblackshear/frigate#2402] [Ask A Question]
I'm using my 4k h265 stream for detection and sub stream (640x480)  for recording and streaming. All works well, but I would like to send a high res image image as a notification. Is there any way to use a different inputs for recordings and snapshots?

**Comment by ozett** [comment]
there are some solutions in the issues section. 
may you have a look there?
example: https://github.com/blakeblackshear/frigate/issues/2199
